### DISCUSSION TITLE
RN 76.2 - No Firebase App '[DEFAULT]' has been created - call firebase.initializeApp()

### DISCUSSION THREAD

**Discussion by despinozakusjanovic** [invertase/react-native-firebase#8170] [Q&A]
Error al obtener el token FCM: [Error: No Firebase App '[DEFAULT]' has been created - call firebase.initializeApp()] [Component Stack]

Hello all,
I have tried many ways, but this error does not go away:

**No firebase app '[DEFAULT]' has been created - call firebase.initializeApp()**

**I am using react-native 76.2 and @react-native-firebase 21.6.1 library.**

I have seen someone else with the same error in 2023, but for my version of RN I have not been able to solve it.

I followed all the steps in the documentation.
I even started it manually on my startup.
I added the Google service and edited build.gradle and app/build.gradle.

```
        buildToolsVersion = "35.0.0".
        minSdkVersion = 24
        compileSdkVersion = 35
        targetSdkVersion = 34
        ndkVersion = "26.1.10909125"
        kotlinVersion = "1.9.24"
```

**Comment by mikehardy** [comment]
Hi there!
Going to need a reproducer on this one, because I have a reproducer for react-native 0.76.x (currently 0.76.3) and it's fine https://github.com/mikehardy/rnfbdemo/blob/main/make-demo.sh
